### PR TITLE
Create README file

### PR DESCRIPTION
**Pull request by nathaliebru**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Install Node Package Manager (NPM)
+
+```bash
+brew install node
+```
```
Comment (fabianovangool) [inline README.md:32]: You can also write multiple commands in one "bash field" with #instructions in between, like this:

```bash
#Install the lastest version of Homebrew
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

#Install Node Package Manager (NPM)
brew install node

#etc 
brew something
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Replace the `scripts` section of your `package.json` with the following:
+
+```bash
+"scripts": {
+  "dev": "next",
+  "build": "next build",
+  "start": "next start"
+}
+```
```
Comment (fabianovangool) [inline README.md:54]: I think this is already covered by your package.json and will be automatically installed by just npm i

File: README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+## Usage
+
+---
+
+To add a new time entry click on the button 'New time entry'. Enter the name of the client, work activity, date and time. Click on the 'Add time entry' button to send this to the server. Your input will be visible on the Timesheets page.
+
+To filter clients use the filter drop-down menu at the top of the timesheets.
+
+To add new team members, this is the same workflow as time entries. Go to the team members page. Click on the 'New Humanoid' button. Enter first name, last name, email address, label, client, role and starting date. Click on the 'Add Humanoid' button to add a new member.
```
Comment (fabianovangool) [inline README.md:71]: You can probably shorten this a bit by combining the add instruction and leaving out the filter options.

**Review by fabianovangool — CHANGES_REQUESTED** [review]

**Review by hans-acc — APPROVED** [review]
🎩  toppertje

**Review by fabianovangool — APPROVED** [review]

**Pull request merged**